### DISCUSSION TITLE
Show Icon Only in New-ConditionalFormattingIconSet

### DISCUSSION THREAD

**Discussion by normansolutions** [dfinke/ImportExcel#1340] [Q&A] [answered]
Firstly, thanks for this package - it's absolutely superb and incredibly useful.

Quick question - is it possible to only show the icon (as against icon and value) when configuring New-ConditionalFormattingIconSet?

Thanks

**Comment by dfinke** [comment]
Please post an example

**Reply by normansolutions** [reply]
Thanks @dfinke 

Attached are three screenshots showing:

* Excel with iconset value
* The inbuilt Excel option to remove iconsset values
* Ideal requirement of not showing iconset value

Below is a snippet of the code I'm using to produce the spreadsheet.

<pre>
$ExcelExportParams = @{
    TitleException = 'Exceptions Report'
    TableName      = 'Exceptions'
    TableStyle     = 'Medium17'
    Path           = Join-Path $logPath "ExcelTest_$(Get-Date -Format 'dd-MM-yyyy').xlsx"
}
$ConditionalFormatConfig = $(
    New-ConditionalFormattingIconSet -Range C:E -ConditionalFormat ThreeIconSet -IconType Symbols2
)
$ColumnResultsAll | Export-Excel $ExcelExportParams.Path -WorksheetName $ExcelExportParams.TitleException `
    -Title $ExcelExportParams.TitleException -TableStyle $ExcelExportParams.TableStyle `
    -ClearSheet -TitleBold -AutoSize -ConditionalFormat $ConditionalFormatConfig
</pre>

Many thanks


![01_WithValue](https://user-images.githubusercontent.com/4348457/206470693-dd1a760d-9cd7-4265-998a-def57bda5861.png)
![02_Option](https://user-images.githubusercontent.com/4348457/206470717-3a17a748-8f3d-48ad-95bc-87f20a9134b9.png)
![03_WithNoValue](https://user-images.githubusercontent.com/4348457/206470747-dfe39743-9355-438a-9510-b6ff7a4dfc08.png)

**Comment by dfinke** [comment, marked as answer]
Thanks for posting. `New-ConditionalFormattingIconSet` does not have a `-ShowValues` option to turn that off. 
Here is a way to get at that setting.  The `$excel.Sheet1.ConditionalFormatting` is array, so you need to correctly identify the one or more you want to turn off.

It'd be easier if you could set this when you do the `New`. I need to think about that.

```powershell
$data = ConvertFrom-Csv @"
Region,State,Units,Price,InStock
West,Texas,927,923.71,1
North,Tennessee,466,770.67,0
East,Florida,520,458.68,1
East,Maine,828,661.24,0
West,Virginia,465,053.58,1
North,Missouri,436,235.67,1
South,Kansas,214,992.47,1
North,North Dakota,789,640.72,0 
South,Delaware,712,508.55,1
"@

$xlfile = "$PSScriptRoot\test.xlsx"
Remove-Item $xlfile -ErrorAction SilentlyContinue

$cfi = New-ConditionalFormattingIconSet -Range E:E -ConditionalFormat ThreeIconSet -IconType Symbols2

$excel = $data | Export-Excel $xlfile -AutoSize -ConditionalFormat $cfi -PassThru

$excel.Sheet1.ConditionalFormatting[0].ShowValue = $false

Close-ExcelPackage $excel -Show
```

**Reply by normansolutions** [reply]
Fantastic - thanks for the tip @dfinke I'll have a dig into your suggestion.

All the best

**Comment by dfinke** [comment]
Add -ShowOnlyIcon to New-ConditionalFormattingIconSet - https://github.com/dfinke/ImportExcel/pull/1341
